### DISCUSSION TITLE
2023-10-06

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#619] [daily_meeting]
### 오늘의 할 일
- 프론트엔드
  - 로그아웃 API 구현

### 개인 일정
- 노아 (5-6 근로)

### 나중에 할 일
- 다음주 월요일(한글날)까지 폴링 구현 완료
- 회원 탈퇴(폴링, 배포 이후에)

**Comment by aak2075** [comment]
### 체크인

10점 - 불금입니다.

### 체크아웃

10점 - 오늘 뭐한거죠

**Comment by jaehee329** [comment]
### 체크인
8점 - 드디어 출근했네요 집에 있으니까 공부를 안해서 진짜 끝나기 전에 취업해야겠다는 생각을 했어요

### 체크아웃
7점 - 할게 많아요 ㅠㅠ

**Comment by yeopto** [comment]
### 체크인

10점 - 간만에 캠퍼스오니 좋네요~

### 체크아웃

10점 - 금욜 밤이라 기분이 좋아요

**Comment by woosung1223** [comment]
### 체크인
9점 - 오랜만에 캠퍼스 오니 활기가 도네요

### 체크아웃
9점 - 이제 다시 본가 갈 생각에 좋네요 굿

**Comment by nlom0218** [comment]
### 체크인

10점 - 캠퍼스 너무 좋아요 

### 체크아웃

9점 - 시간이 너무 빨라요 😭

**Comment by woo-jk** [comment]
### 체크인
10점 - 월요일 같은데 금요일 ㄷㄷㄷ

### 체크아웃
10점 - 할게 너무 많은데 시간이 없어요. 주말 달려야겠습니다

**Comment by MoonJeWoong** [comment]
### 체크인 
9점 - 캠퍼스 오니까 좋네요 ㅎㅎ... 근데 뭐했다고 오늘이 금요일이죠... ㅠㅠ

### 체크아웃
9점 - 이제 뭐 기능 개발 끝났으니 취준 들어가나요?
